requirements-dev.txt

```diff
@@ -4,10 +4,9 @@ packmol>=1!18.013
 nglview>=0.6.2.3
 oset
 parmed
-# TODO: Remove pinning of mdtraj version once mol2 reader issues have been resolved.
-#       https://github.com/mdtraj/mdtraj/pull/1378
-#       Once this has been released in a new MDTraj version, this can be removed.
-mdtraj==1.9.1
+mdtraj=1.9.1;python_verion="3.5"
```

Comment:
is it supposed to be: `mdtraj=1.9.1` or `mdtraj==`?

Reply:
To be honest I'm not sure, since this mixture of `pip` and `conda` behavior has me confused. But the stack page I linked uses `==` so we can try that.

Reply:
nope
```
$ conda create -n test-environment python=$PYTHON_VERSION --file requirements-dev.txt
InvalidVersionSpec: Invalid version '1.9.1;python_verion': invalid character(s)
The command "conda create -n test-environment python=$PYTHON_VERSION --file requirements-dev.txt" failed and exited with 1 during .
Your build has been stopped.
```